string launch_command
---
string result
